src/FishyFlip/Models/SubscribeRepoMessage.cs

```diff
@@ -69,6 +69,11 @@ public class SubscribeRepoMessage
     /// </summary>
     public FrameIdentity? Identity { get; internal set; }
 
+    /// <summary>
```

Comment:
This is `SubscribeRepoMessage` where `Labels` doesn't exist.

I get why you put it here (It overlaps with the existing object) but this should probably be moved out into its own object and parsed that way.